python/pyspark/pandas/numpy_compat.py

```diff
@@ -223,11 +223,16 @@ def maybe_dispatch_ufunc_to_spark_func(
             op_name
         )
 
+        if is_remote():
+            from pyspark.sql.connect.column import Column as ConnectColumn
+
+            Column = ConnectColumn
+        else:
+            Column = PySparkColumn  # type: ignore[assignment]
+
         @no_type_check
         def convert_arguments(*args):
-            args = [
-                F.lit(inp) if not isinstance(inp, (Column, ConnectColumn)) else inp for inp in args
-            ]
+            args = [F.lit(inp) if not isinstance(inp, Column) else inp for inp in args]
```

Comment:
This one, I actually don't think we should check Column condition because `lit` returns `Column` as is.